Makefile

```diff
@@ -93,7 +93,7 @@ push-gitea-image:
 push-guacamole-image:
 	$(call push_image,"guac-server","./templates/workspace_services/guacamole/version.txt")
 
-tre-deploy:
+tre-deploy: tre-start
```

Comment:
So, `tre-deploy` will always first try and start the TRE now?

Reply:
Yes, that' right. The reason is that if the TRE is "stopped" then TF will fail with weird errors